source/dataman/ZfpMan.h

```diff
@@ -24,7 +24,7 @@ class ZfpMan : public CompressMan
     virtual void flush();
     void *compress(void *p_data, json &p_jmsg);
```

Comment:
compress and decompress should have comments that document the memory they return needs to be `free`d by the caller else it will leak.